File: Python/1.py
class Carro:
    def __init__(self, marca,modelo,ano):
        self.marca = marca
        self.modelo = modelo
        self.ano = ano

    def exibir(self):
        print(
            self.marca,
            self.modelo,
            self.ano
        )


carro1 = Carro('Lamborghini', 'Temerario', 2024)
carro2 = Carro('Fiat', 'Strada', 2020)

carro1.exibir()
carro2.exibir()

File: Go/1/1.go
package main

import "fmt"

type Carro struct {
    marca  string
    modelo string
    ano    int
}
func main() {
    veiculo1 := Carro{"Lamborghini", "Temerario", 2024}
    veiculo2 := Carro{"Fiat", "Strada", 2015}
    veiculo3 := Carro{"Chevrolet", "Onix", 2019}
   
    fmt.Println(veiculo1)
    fmt.Println(veiculo2)
    fmt.Println(veiculo3)
}